src/layers/PathDetailPopup.module.css

```diff
@@ -0,0 +1,37 @@
+/* todo: this is just copy-pasta from https://openlayers.org/en/latest/examples/popup.html, I have no idea what all this means,
+   and the popup looks ugly still, at the very minimum the background color needs to be set though
+   the popup design needs an overhaul anyway... https://github.com/graphhopper/graphhopper-maps/issues/136
+   */
```

Comment:
Same as for the context menu

Reply:
Ok. And we have this issue already: https://github.com/graphhopper/graphhopper-maps/issues/136